.github/workflows/maven.yml

```diff
@@ -27,11 +27,21 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
       - name: Build with Maven
-        run: mvn -s .github/workflows/mvn-settings.xml -B package --file tools/pom.xml -DskipTests=true
+        run: >
+          mvn -s .github/workflows/mvn-settings.xml -B package --file tools/pom.xml
+          -DskipTests=true
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Setup MySQL
+        run: |
+          sudo systemctl start mysql.service
+          sleep 3
+          mysql --user=root --password=root < tools/cldr-apps/test-setup.sql
       - name: Test with maven
-        run: mvn -s .github/workflows/mvn-settings.xml -B test --file tools/pom.xml
+        run: >
+          mvn -s .github/workflows/mvn-settings.xml -B test --file tools/pom.xml
+          '-Dorg.unicode.cldr.unittest.web.jdbcurl=jdbc:mysql://cldrtest:VbrB3LFCr6A!@localhost/cldrtest?useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC'
```

Comment:
What's VbrB3LFCr6A!
?

Reply:
I see now it's in test-setup.sql. Is there any security risk in having that in git? And, could we define it once and name it, rather than repeating it in these two files?

Reply:
@btangmu There isn't a risk, because that user is only setup within the git envinronment.

Reply:
@btangmu I captured your comment in https://unicode-org.atlassian.net/browse/CLDR-8383 - can we merge this (not a regressin in this respect) and fix in the future?)